src/app_handle.rs

```diff
@@ -353,7 +392,34 @@ impl ApplicationHandle {
             return;
         };
         let window_id = window.id();
-        let window_handle = WindowHandle::new(window, view_fn, transparent, apply_default_theme);
+
+        #[cfg(target_arch = "wasm32")]
+        {
+            let canvas = window.canvas();
```

Comment:
I'm wondering if it's better to provide winit with the canvas, so that we can rely on the browser to set the size of it.

Reply:
Done.